### PR TITLE
Php 7.1

### PR DESCRIPTION
**Pull request by bkuhl**
_rolls dice for dockerhub credentials_

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ for version in "${cli[@]}"; do
 done
 
 echo "Building base cli images: ${tagscli}"
-docker build ${tagscli} -f 7.0/Dockerfile .
+docker build ${tagscli} -f debial-jesse/cli/Dockerfile .
```
Comment (jmauerhan) [inline build.sh]: is `debial-jesse` a valid image? I thought it was debian?

Comment (bkuhl) [inline build.sh]: Woah, terrible typo

Comment (bkuhl) [inline build.sh]: Fixed

File: README.md [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
 ## Supported tags
-- [`7.0.10-cli`, `7.0-cli`, `7-cli`, `cli` (*7.0/Dockerfile*)](https://hub.docker.com/r/realpage/php/) 
-[![Image Layers](https://img.shields.io/imagelayers/layers/realpage/php/7.0-cli.svg)](https://imagelayers.io/?images=realpage/php:7.0-cli)
-- [`7.0.10-fpm`, `7.0-fpm`, `7-fpm`, `fpm` (*7.0/fpm/Dockerfile*)](https://hub.docker.com/r/realpage/php/) 
-[![Image Layers](https://img.shields.io/imagelayers/layers/realpage/php/7.0-fpm.svg)](https://imagelayers.io/?images=realpage/php:7.0-fpm)
-- [`7.0.10-alpine`, `7.0-alpine`, `7-alpine`, `alpine` (*7.0/alpine/Dockerfile*)](https://hub.docker.com/r/realpage/php/) 
-[![Image Layers](https://img.shields.io/imagelayers/layers/realpage/php/7.0-alpine.svg)](https://imagelayers.io/?images=realpage/php:7.0-alpine)
-- [`7.0.10-fpm-alpine`, `7.0-fpm-alpine`, `7-fpm-alpine`, `fpm-alpine` (*7.0/fpm/alpine/Dockerfile*)](https://hub.docker.com/r/realpage/php/) 
-[![Image Layers](https://img.shields.io/imagelayers/layers/realpage/php/7.0-fpm.svg)](https://imagelayers.io/?images=realpage/php:7.0-fpm-alpine)
+- [`7.1.0-cli`, `7.1-cli`, `7-cli`, `cli` (*debian-jesse/cli/Dockerfile*)](https://hub.docker.com/r/realpage/php/)
```
Comment (jmauerhan) [inline README.md]: jesse or jessie?

**Comment by bkuhl** [comment]
Since I changed the directory structure, my intention is to tag `master` with `7.1.0` after this is merged and maintain versioning that way instead of by directory structure.

**Pull request merged**